### PR TITLE
PGD Quickstart link fixes, TPA option change

### PR DESCRIPTION
**Pull request by josh-heyer**
## What Changed?

Missed a few places during yesterday's update.

### CODE REVIEW

File: product_docs/docs/pgd/5/tpa.mdx [hunk 1]
```diff
@@ -51,7 +51,7 @@ The available configuration options include:
 | `--location-names l1 l2 l3` | Required. Specifies the number and name of the locations PGD will be deployed to. |
 | `--data-nodes-per-location N` | Specifies number of data nodes per location. Default 3. |
 | `--add-witness-node-per-location` | For even number of data nodes per location, this will add witness node to allow for local consensus. This is enabled by default for 2 data node locations. |
-| `--cohost-proxies` | Whether to put PGD-Proxies to data nodes. By default proxies are installed to separate hosts |
+| `--add-proxy-nodes-per-location` | Whether to separate PGD-Proxies from data nodes, and how many to configure. By default one proxy is configured and cohosted for each data node |
```
[the remark this answers is not in the document]
Comment (djw-m) [reply product_docs/docs/pgd/5/tpa.mdx:54]: Also no entry for the now mandatory flavour and version fields.

**Review by djw-m — COMMENTED** [review]
The fixes look fine for the quickstarts but the TPA page needs a bit more love. Also noticed it references "tpcaexec(sic) info" as reporting what values are available but tpaexec info only reports back tpaexec's env vars. Was that meant to be `tpaexec configure --architecture PGD-Always-ON --help` for all the values?

File: product_docs/docs/pgd/5/tpa.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (josh-heyer) [reply product_docs/docs/pgd/5/tpa.mdx:54]: [That is what it does:](https://www.enterprisedb.com/docs/tpa/latest/architecture-PGD-Always-ON/)

> To create separate PGD-Proxy instances instead, use --add-proxy-nodes-per-location 3 (or however many proxies you want to add).

Adding `--addproxy-nodes-per-location 2` to the invocation produces a configuration akin to those produced prior to 23.16 (3 data nodes, 2 separate proxy nodes)

**Comment by djw-m** [comment]
Hmm... not hinted at by the help then....

  --cohost-proxies      Not needed; pgd-proxy runs on the data nodes by
default
  --add-proxy-nodes-per-location PROXY_NODES_PER_LOCATION
                        Number of separate PGD-Proxy nodes to add in each
location

Especially with the "Not needed" implying it always runs pgd_proxy on the
data nodes.

**Comment by djw-m** [comment]
Bonus... specify --add-proxy-nodes-per-location 2 --cohost-proxies

and the --add-proxy-nodes-per-location is completely ignored - you only get
cohosted proxies

File: product_docs/docs/pgd/5/tpa.mdx (hunk 1 above)
[the remark this answers is not in the document]
Comment (josh-heyer) [reply product_docs/docs/pgd/5/tpa.mdx]: Updated with a (short) summary of flavour options and link into TPA docs for more. Corrected the bogus reference to info.

**Review by drothery-edb — APPROVED** [review]

**Pull request merged**